docs/guides/prefect-deploy.md

```diff
@@ -883,6 +890,10 @@ These deployments can be managed independently of one another, allowing you to d
     Prefect supports multiple deployment declarations within the `prefect.yaml` file. This method of declaring multiple deployments allows the configuration for all deployments to be version controlled and deployed with a single command.
 
     New deployment declarations can be added to the `prefect.yaml` file by adding a new entry to the `deployments` list. Each deployment declaration must have a unique `name` field which is used to select deployment declarations when using the `prefect deploy` command.
+    
+    !!! warning
+        When using a custom `prefect.yaml` file, remember that the value for deployment `entrypoints` must be relative to 
```

Comment:
```suggestion
        When using a custom `prefect.yaml` file, remember that the value for the deployment `entrypoint` must be relative to 
```

Reply:
since I don't believe there is such a thing as `entrypoints` key (to my knowledge!), didn't want to confuse the user